CHANGELOG.md

```diff
@@ -6,11 +6,40 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [[Unreleased]]
+
+## [2.0.0]
+### Added:
+- Wallet support for regular key compatibility
+- Added new ways of wallet generation: `from_seed`, `from_secret`, `from_entropy`, `from_secret_numbers`
+- Added address alias to `Wallet.classic_address`
+- Replaced `Wallet.classic_address` with `Wallet.address` to avoid confusion. (`classic_address` is the same as your XRPL account `address`, and is only called classic since it's an older standard than `x-address`)
+
+### Changed:
+- Updated params for Wallet class constructor
```

Comment:
This is missing network id changes being mentioned.

Reply:
Added :)